docs/framework/configure-apps/file-schema/runtime/enablelegacyxmlsettings-element.md

```diff
@@ -0,0 +1,52 @@
+---
+title: EnableLegacyXmlSettings element
+ms.date: 12/08/2020
+---
+# \<EnableLegacyXmlSettings> Element
+
+Specifies whether to use XML parsing settings from .NET Framework 4.5.1 and earlier versions.
+
+[**\<configuration>**](../configuration-element.md)\
+&nbsp;&nbsp;[**\<runtime>**](runtime-element.md)\
+&nbsp;&nbsp;&nbsp;&nbsp;**\<EnableLegacyXmlSettings>**
+
+## Syntax
+
+```xml
+<EnableLegacyXmlSettings enabled="0"|"1" />
```

Comment:
I think the switch has to be set in the registry. 

https://referencesource.microsoft.com/#System.Xml/System/Xml/Core/XmlReaderSettings.cs,747